80s Game/Assets/Scripts/Multiplayer/PlayerInputWrapper.cs

```diff
@@ -87,13 +88,15 @@ private void OnMove(InputValue value)
             snailModifier = 0.5f;
         }
 
-
         Vector2 adjustedInput = Vector2.Scale(value.Get<Vector2>(), sensitivity * config.sensitivity * snailModifier);
         if (isFlipped)
         {
             adjustedInput *= -1;
         }
-        player.HandleMovement(adjustedInput * Time.deltaTime);
+
+        // Mouse deltas are already a displacement-based value, so lag-spikes
+        // will cause the cursor to jump around (No Time.deltaTime adjustments)
+        player.HandleMovement(adjustedInput / 150f);
```

Comment:
any chance we can make that 150f a non-magic number?

Reply:
indeed